components/fctl/pkg/clients.go

```diff
@@ -45,6 +46,19 @@ func MembershipServerInfo(ctx context.Context, client *membershipclient.APIClien
 	return serverInfo.Version
 }
 
+func ValidateMembershipServerVersion(ctx context.Context, client *membershipclient.APIClient, version string) error {
+	serverVersion := MembershipServerInfo(ctx, client)
+	if !semver.IsValid(serverVersion) {
+		return nil
+	}
+
+	if semver.Compare(version, serverVersion) >= 0 {
```

Comment:
I have a doubt about this check: you're checking that serverVersion < 0.27.1 to accept ?